Makefile.toml

```
@@ -204,24 +204,22 @@ description = "Defines the E2E-tests flow."
 workspace = false
 dependencies = [
     "build",
-    "pre-dry-tests",
+    "install_test_dependencies",
     "dry-tests"
 ]
 
 [tasks.dry-tests]
 description = "Runs tests that don't require any external services"
 workspace = false
 private = true
-script_runner = "bash"
-script = [
-'''
-export ROOT=$(pwd);
-./api_tests/npm_wrapper.sh ./dry/
-'''
-]
+install_script = [ "( cd ./api_tests && yarn install; )" ]
+script = [ "./api_tests/harness.js ./api_tests/dry/*.js" ]
 
-[tasks.pre-dry-tests]
-alias = "pre-e2e-tests"
+[tasks.install_test_dependencies]
```

```suggestion
[tasks.install-test-dependencies]
```
We use kebab-case for task names